packages/client/hmi-client/src/assets/css/theme/_variables.scss

```diff
@@ -947,4 +947,5 @@ $petri-inputBox: var(--surface-0);
 	--node-header-hover: var(--text-color-subdued);
 	--z-index-modal: 1000;
 	--keyboard-zindex: calc(var(--z-index-modal) + 100);
+	--explorer-background-colour: #F4F7FA;
```

Comment:
Could you check which `--surface` (or `--grey`) value is the closest to it and use that instead?

Reply:
Ya I can change it to one of those - I was going with the Figma design colours.  Should we use these defaults instead?

Reply:
Went with `surface-100` in the meantime